app/views/application_view.rb

```diff
@@ -59,7 +59,7 @@ def head_links
   end
 
   def fb_campaigns_on
-    true
+    true unless Rails.env.test?
```

Comment:
What's this?

Reply:
This was to stop facebook errors in test. we should do this on develop probably, so I will remove it from this PR as it's not specifically related.